packages/addons/sveltekit-adapter/index.ts

```diff
@@ -80,35 +80,26 @@ export default defineAddon({
 			}
 
 			const { value: config } = exports.createDefault(ast, { fallback: object.create({}) });
+
+			// remove leading comments from the `adapter` property
 			const kitConfig = config.properties.find(
```

Comment:
it might be a bit of a drive-by refactor if done in this PR, so may justify a follow up PR - but its probably a good idea to now update the various `config.properties.find(...)` usages to use `object.property`